src/goController.php

```diff
@@ -557,6 +557,14 @@ private function handleException(Exception $e) {
                 $heading = 'Random Alias Error.';
                 $msg = '<p>'. $e->getMessage() . '</p>';
                 break;
+            case lilURL::ERR_MALICIOUS_URL:
+                    $heading = 'The URL you submitted has been deemed malicious.';
+                    $msg = '
+                        <p>
+                            If you think this is an error reach out to <a href="mailto:wdn@unl.edu">wdn@unl.edu</a>
```

Comment:
is [wdn@unl.edu](mailto:wdn@unl.edu) an active email? maybe? does anyone (bob) look at it? usually we would direct people to [dxg@lists.nebraska.edu](mailto:dxg@lists.nebraska.edu)